source/reference/combining.rst

```diff
@@ -15,46 +15,27 @@ that and are described in `structuring`.  Combining schemas may be as
 simple as allowing a value to be validated against multiple criteria
 at the same time.
 
-For example, in the following schema, the ``anyOf`` keyword is used to
-say that the given value may be valid against any of the given
-subschemas.  The first subschema requires a string with maximum
-length 5. The second subschema requires a number with a minimum value
-of 0.  As long as a value validates against *either* of these schemas,
-it is considered valid against the entire combined schema.
-
-.. schema_example::
-
-    {
-      "anyOf": [
-        { "type": "string", "maxLength": 5 },
-        { "type": "number", "minimum": 0 }
-      ]
-    }
-    --
-    "short"
-    --X
-    "too long"
-    --
-    12
-    --X
-    -5
+These keywords correspond to well known boolean algebra concepts like
+AND, OR, XOR, and NOT. You can often use these keywords to express
+complex constraints that you can't otherwise express with standard
```

Comment:
```suggestion
complex constraints that can't otherwise be express with standard
```

Reply:
I agree, with the change, but it's missing something.

```suggestion
complex constraints that can't otherwise be expressed with standard
```